src/app/components/search/CustomSliderComponent.tsx

```diff
@@ -0,0 +1,145 @@
+import HorizontalRuleIcon from "@mui/icons-material/HorizontalRule";
+import { Box, Slider, Stack, Typography } from "@mui/joy";
+import React, { useEffect } from "react";
```

Comment:
Don't import React, just import the hooks and anything else you need from the library